connector/connect/src/main/protobuf/spark/connect/base.proto

```diff
@@ -183,6 +183,46 @@ message ExecutePlanResponse {
   }
 }
 
+// Base request to execute Catalog operations.
+message CatalogRequest {
```

Comment:
will this message also support commands like `createTable`, `dropTempView`?

Reply:
I think the only exception case is `createTabe` because it returns a DataFrame.... My understanding is `createTable` should go to `ExecutePlan` rpc, same as other DataFrame API. Because  one could use `createTable` to chain the other DataFrame API.

other API can go through there.